config/config.go

```diff
@@ -83,8 +83,10 @@ func (b *OverMaxRetryBehavior) UnmarshalText(text []byte) error {
 	return nil
 }
 
-var OverMaxRetryBehaviorAlive = "alive"
-var OverMaxRetryBehaviorDie = "die"
+const (
+	OverMaxRetryBehaviorAlive = "alive"
```

Comment:
__[golint]__
* [golint] exported const OverMaxRetryBehaviorAlive should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->